sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ReceiverOptions.java

```diff
@@ -11,10 +11,16 @@
 class ReceiverOptions {
     private final ReceiveMode receiveMode;
     private final int prefetchCount;
+    private final String sessionId;
 
     ReceiverOptions(ReceiveMode receiveMode, int prefetchCount) {
+        this(receiveMode, prefetchCount, null);
```

Comment:
Why do we need another constructor? This is only used in SBCB to pass to the receiver.